app/src/test/java/io/apicurio/registry/ConfluentCompatApiTest.java

```diff
@@ -218,14 +218,25 @@ public void testDisabledStateCheck() throws Exception {
            .then()
                .statusCode(204);
         
-        // GET - shouldn't return as the state has been changed to DISABLED
+        // GET - object should return when the state has been changed to DISABLED
         TestUtils.retry(() -> {
             given()
                 .when()
                     .contentType(ContentTypes.COMPAT_SCHEMA_REGISTRY_STABLE_LATEST)
                     .get("/ccompat/subjects/{subject}/versions/{version}", SUBJECT, "latest")
                 .then()
-                    .statusCode(400);
+                    .statusCode(200)
+                    .body("schema", equalTo(SCHEMA_SIMPLE));
+        });
```

Comment:
I'm not 100% sure about this.  There's no concept of "state" in the ccompat API, right?  And the entire schema is available (obviously) from the `schema` property of the result.  It seems like maybe this should return a 404?

Reply:
I wasn't sure about that either. You're right that the ccompat API doesn't have state, so I have updated to return a 404 when users GET a disabled artifact on the `/ccompat/subjects/{subject}/versions/{version}` endpoint.